files/zh-cn/web/html/element/iframe/index.md

```diff
@@ -60,7 +60,7 @@ slug: Web/HTML/Element/iframe
 
 - `sandbox`
 
-  - : 该属性对呈现在 iframe 框架中的内容启用一些额外的限制条件。属性值可以为空字符串（这种情况下会启用所有限制），也可以是用空格分隔的一系列指定的字符串。有效的值有：
+  - : 该属性对呈现在 iframe 框架中的内容启用一些额外的限制条件。属性值可以为空字符串（这种情况下会启用所有限制），也可以是用空格分隔的一系列指定的字符串（将移除特定的限制）。有效的值有：
```

Comment:
Let's sync the content with the [en-US document](https://github.com/mdn/content/blob/e194ecd391cc18ac9eef0648abcaa49bad4c788c/files/en-us/web/html/element/iframe/index.md?plain=1#L74)

```suggestion
  - : 控制应用于嵌入在 `<iframe>` 中的内容的限制。该属性的值可以为空以应用所有限制，也可以为空格分隔的标记以解除特定的限制：
```